Bindings/JS/src/index.ts

```diff
@@ -12,7 +16,9 @@ export default class Lazer {
         return instance;
     }
 
-    async computeDifficulty(beatmapContent, rulesetId, mods) {
+    async computeDifficulty(beatmapContent: string, rulesetId: number, mods: number): Promise<number> {
        return await this.osu.ComputeDifficulty(beatmapContent, Number(rulesetId), mods);
     }
+
+    private osu!: Osu;
```

Comment:
Is there anything special with the `!` here? Would remove if not, because it looks like a typo given the above `this.osu` call.

Reply:
that `!`  is an not-null assertion, telling the compiler that while the field doesn't get initialized in the constructor, it should assume that it's not null or undefined when it's being used.
I can refactor it to initialize the field in the constructor instead

Reply:
Aha, all good then. I'm used to the C# way of doing it so I didn't know the syntax :)